dts/arm/nxp/nxp_rt118x.dtsi

```diff
@@ -44,6 +44,13 @@
 			};
 		};
 	};
+
+	soc {
+		temp: temp {
+			compatible = "nxp,tmpsns";
```

Comment:
Need to add base address and interrupt properties. Also this should be part of the peripheral block